python/lsst/daf/butler/core/dimensions/_records.py

```diff
@@ -287,6 +287,15 @@ def __reduce__(self) -> tuple:
         mapping = {name: getattr(self, name) for name in self.__slots__}
         return (_reconstructDimensionRecord, (self.definition, mapping))
 
+    def _repr_html_(self) -> str:
+        r"""Call (called automatically) when displaying in IPython/notebooks.
+
+        If one types the name of a record object and just hits return, the
```

Comment:
I think this text is trying to say what happens when this method doesn't exist -- I think I'd rather it said what this method does.

```
Overrides the default representation of these objects in IPython/Jupyter giving a more
readable output that understands embedded newlines.
```
or something.

Reply:
I dislike this as much as you do - this was my 5th or so attempt, because D401 meant I was getting a linter error for not putting it in the imperative, but wanting to get across that one doesn't actually call this method directly, but is called from by the ipython/notebook kernel automatically.

I guess I can use `Override` instead of `Override` and chop down to a single line to make something comply here, but it's going to be pretty gross whatever I do in order to comply with the rules.

Reply:
I was commenting on the second paragraph, not the summary line.